docs/api/instruments/agilent/agilentE4418B.rst

```diff
@@ -0,0 +1,7 @@
+##############################################
+Agilent E4418B/E4419B Microwave Power meter
+##############################################
+
+.. automodule:: pymeasure.instruments.agilent.agilentE4418B
```

Comment:
You can add both instruments and also the Channel in order to have a full documentation of the devices.